stdpopsim/genomes.py

```diff
@@ -309,21 +312,23 @@ def species_contig(
     def length(self):
         return self.recombination_map.sequence_length
 
-    @property
     def all_intervals_array(self):
         """
-        Returns an (n, 3) numpy array with the intervals across all genomic
-        element types in the form (left, right, type).
+        Returns a dictionary with two things: "intervals" is the (n, 2)
+        array of all intervals across all DFEs, and "dfes" is a length-n
+        array of integer labels, so that
+        intervals[i] is tagged by the DFE dfe_list[dfes[i]].
```

Comment:
I think it's conceptually confusing to bundle together the two semantically different things (start/endpoints, then labels) even if they're all integers, so I changed the output format.

Reply:
Not sure if I've got it right, though - is it easy to put this in a pandas data frame, for instance?